skyfield/sgp4lib.py

```diff
@@ -153,6 +153,30 @@ def ITRF_position_velocity_error(self, t):
         rITRF, vITRF = TEME_to_ITRF(t.ut1, rTEME, vTEME)
         return rITRF, vITRF, error
 
+    def is_sunlit(self, ephemeris, times):
+        """Calculates if the satellite is currently sunlit (True/False) at each time in times. Returns an array of booleans."""
+        Re = 6378.137  # Equatorial radius of the Earth in km
```

Comment:
Instead of hard-coding this value, go ahead and use `ERAD` from the `skyfield.constants` module.